src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -121,6 +124,7 @@ public enum UnreferencedTypesHandlingOption
         public static bool SingleTopLevelClient { get; private set; }
         public static bool SkipSerializationFormatXml { get; private set; }
         public static bool DisablePaginationTopRenaming { get; private set; }
+        public static bool ArrayToPage { get; private set; }
```

Comment:
I don't think this will be a bool.  It will be specific to an individual operation.  It will most likely be a decorator in cadl that only the dotnet emitter will care about.

Reply:
like `@PageResponse(boolean)`?

How could we not confuse other languages?